build/configs/imxrt1050-evk/uvc_camera/Make.defs

```diff
@@ -0,0 +1,135 @@
+###########################################################################
```

Comment:
can you add samsung license here?

Reply:
Base is hello , same is the status there.  Not sure is there any reason for it.